internal/consensus/pbts_test.go

```diff
@@ -612,11 +607,12 @@ func TestPBTSEnableHeight(t *testing.T) {
 				proposal.Timestamp = time.Now()
 			}
 			signProposal(t, proposal, chainID, vss[proposer])
-			cs.SetProposalAndBlock(proposal, block, blockParts, "p")
+			err := cs.SetProposalAndBlock(proposal, block, blockParts, "p")
+			require.NoError(t, err)
 			ensureProposal(proposalCh, height, round, blockID)
 		}
 
-		delta := time.Now().Sub(block.Time)
```

Comment:
this should not be reverted

Reply:
It's before my PR, so we're not "needing" `cmttime.Now` yet. I'll be fixing this in my PR, when I rebase it on top of this